### ISSUE TITLE
Cannot set default type in Schema

### ISSUE THREAD

**Issue by cbanbury** [drew-y/meteor-autoform-datetimepicker#3]
Hi, more of a question than an issue, but having some trouble trying to set the default type as shown the the README: 

```
var Schema = {};

Schema.Date = new SimpleSchema({
  date: {
    type: String,
    autoForm: {
      type: 'datetimepicker'
    }
  }
});

Schema.test = new SimpleSchema({
  my_date: {
    type: Date
  }
});

MyCollection.attachSchema(Schema.test);
```

This gives the following error: 

```
Error: Invalid definition for date field.
    at packages/aldeed_simple-schema/packages/aldeed_simple-schema.js:1328:1
```

What am I doing wrong? Thanks in advance

**Comment by cbanbury** [comment]
OK, I've got this working but I had to nest it like this: 

```
Schema.Date = new SimpleSchema({
    date: {
        type: String,
        autoform: {
            afFieldInput: {
                type: "datetimepicker"
            }
        }
    }
});
```

I'm new to using autoform so I don't know if this is a recent change but could you update the documentation if this is required.

**Comment by drew-y** [comment]
That was an error on my part. Thanks for pointing it out. Fixed with 7bd2dfbd7416a4729c5c79f6134b7fd31040a0a4.
